### PR TITLE
[#48862] Add PDF Export of work plans to our documentation

### PR DESCRIPTION
**Pull request by MayaBerd**
https://community.openproject.org/work_packages/48862

### CODE REVIEW

**Review by as-op — APPROVED** [review]

File: docs/user-guide/work-packages/exporting/README.md [hunk 1]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
 
 # Exporting work packages
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: I would suggest we write **Export work packages** (just like create, edit or copy). Also in the title of the document -> Navigation.

File: docs/user-guide/work-packages/exporting/README.md [hunk 2]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
 
-You can export work packages from your OpenProject instance to other formats using the export functionality. 
+You can export work packages from your OpenProject instance in various formats using the export functionality. 
 
 
 
 ## How to trigger an export
 
-To export a work package table or card view to another format, visit the *Work packages* module or a saved view and click on the settings icon in the top right. Trigger the **Export** dialog from the dropdown menu that opens.
+To export a work package table or card view to another format, visit the *Work packages* module or select a saved view, opening a work package table that you want to export. Click on the settings icon in the top right corner. Trigger the **Export** dialog from the dropdown menu.
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: To export a work package table, visit the *Work packages* module and select the list of work packages which you want to export. You can either export a saved view or filter the work packages based on your needs. Click on the settings icon in the top right corner. Trigger the **Export** dialog from the dropdown menu.

File: docs/user-guide/work-packages/exporting/README.md [hunk 3]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
 
-![Exporting from the table](export-gantt-chart.png)
+![Exporting from the table](openproject_export_wp.png)
 
-This will open a dialog similar to the following screenshot, where you can select the desired format. Below, we will detail how to control the data being exported as well as the various formats.
+This will open a dialog where you can select the desired format. Below, we will detail how to adjust which data should be exported as well as what the various formats contain.
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: Click on one of the possible formats to start the export.

File: docs/user-guide/work-packages/exporting/README.md [hunk 4]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
 
-> **Note**: If you need to export more than 500 items, filter the work packages appropriately to export multiple files.
+> **Note**: PDF export options includes all of the work packages in the selected work package table, regardless of the limit. The possible export limit is relevant for XLS, CSV and Atom export options.
 
-For self-hosted versions of OpenProject you can change this limit in the [work package settings](../../../system-admin-guide/manage-work-packages/work-package-settings/) in the system administration.
+**Columns**
 
+The exported file will display the columns that are activated for the work package table. By adding or removing specific columns you can control the columns that will be included into the exported file. Please make sure to [save](../work-package-table-configuration/#save-work-package-views) the work package view you configured for the changes to be included into the report.
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: [save the work package view] --> all as a link?

File: docs/user-guide/work-packages/exporting/README.md [hunk 5]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
 
-**Columns**
+Some formats such as PDF will limit the number of columns available due to limitations of the PDF rendering engine to avoid overflowing the available space.
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: Some spaces and line breaks can be removed.

File: docs/user-guide/work-packages/exporting/README.md [hunk 6]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
 
-Some formats such as PDF will limit the number of columns available due to limitations of the PDF rendering engine to avoid overflowing the available space.
 
+## Export format options
+### PDF export
 
+OpenProject has multiple options of exporting work packages in PDF format:
 
-## PDF export
+- **PDF Table** exports the work package table displaying work packages as single rows.
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md:44]: with the selected columns for the work package table.

File: docs/user-guide/work-packages/exporting/README.md [hunk 7]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: Remove the h3 PDF export and list both PDF options (PDF Table and PDF Report and PDF Report with images) as h3.

File: docs/user-guide/work-packages/exporting/README.md [hunk 8]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
 
-OpenProject has multiple options of exporting the results to PDF:
+![OpenProject PDF Table export](openproject_pdf_table_export.png)
 
+- **PDF Report** includes a table of contents (listing all of the work packages), followed by the description of single work packages in a block form. For each work package a table of attributes is included (attributes correspond to the columns in the work package table).
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md:49]: Add a small introduction, e.g. : With PDF Reports you can export detailed up to date work plans for your project in a nice format. ...It includes ...

Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md:49]: (listing all of the work packages of the selected work package table view)

File: docs/user-guide/work-packages/exporting/README.md [hunk 9]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
-- **PDF with attachments** of work packages with their attachments inlined to fit as best as possible.
-- **PDF with descriptions and attachments** The above options combined to create a PDF export with work package descriptions and their attachments.
+![OpenProject_work_package_table_sum](openproject_wp_table_total_sum.png)
 
+![OpenProject_pdf_report_sum](openproject_wp_report_total_sum.png)
 
+- **PDF Report with images** is the same as PDF Report, but also includes the images from the work package description. 
 
-## Excel (XLS) export
+   > **Note**: Images attached to the work package or in activity comments are not included into exported report.
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: are not included into the exported report.

File: docs/user-guide/work-packages/exporting/README.md [hunk 10]
```diff
@@ -8,76 +8,92 @@ keywords: work package exports, CSV, Excel, XLS, PDF
...
 
-OpenProject can export the table for Microsoft Excel with the following options:
+   ![OpenProject_work_package_export](openproject_pdf_report_images.png)
 
-- **XLS** a plain sheet that matches the OpenProject work packages table with its columns and work packages as rows matching the selected filter(s).
-- **XLS with descriptions** same as above, but with an additional column for work package descriptions, which cannot be selected in the table.
-- **XLS with relations** Same as XLS, but with additional columns to list each work package relation in a separate row with the relation target and its ID and relation type included in the export.
+
+### Excel (XLS) export
```
Comment (birthe) [inline docs/user-guide/work-packages/exporting/README.md]: Can you please move all Excel format options below also to h3.

**Review by birthe — CHANGES_REQUESTED** [review]

**Pull request merged**